addons/mail/static/src/model/record_internal.js

```diff
@@ -203,9 +231,8 @@ export class RecordInternal {
         this.fieldsComputeOnNeed.delete(fieldName);
         let computedValue;
         try {
-            computedValue = Model._.fieldsCompute
-                .get(fieldName)
-                .call(this.fieldsComputeProxy2.get(fieldName));
+            const computeProxy = this.fieldsComputeProxy2.get(fieldName);
+            computedValue = Model._.fieldsCompute.get(fieldName).call(computeProxy, computeProxy);
```

Comment:
new feature: have current record as param of `compute`.

Not useful even for 1st commit (wanted to use arrow functions internally, but in the end used `function` with `this` scope), but this may still be useful for shorter syntax of simple compute using `this` thanks to arrow function.